### PR TITLE
fix(php-cs-fixer): update of PhpCsFixerRuleSet

### PR DESCRIPTION
**Pull request by abuathier**
## Description

Jira: 🏷️ MON-171226

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [X] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [X] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

GitHub CI

#### Community contributors & Centreon team

- [X] I have followed the **coding style guidelines** provided by Centreon
- [X] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [X] I have commented my code, especially **hard-to-understand areas** of the PR.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/src/Core/Domain/Configuration/Model/MetaService.php [hunk 1]
```diff
@@ -30,70 +31,53 @@ class MetaService
...
 
     /**
      * @param int $id
      * @param string $name
      * @param string $calculationType
      * @param int $metaSelectionMode Selection mode for services to be considered for this meta service.
-     *     0 - In service list mode, mark selected services in the options on meta service list.
-     *     1 - In SQL matching mode, specify a search string to be used in an SQL query.
+     *                               0 - In service list mode, mark selected services in the options on meta service list.
+     *                               1 - In SQL matching mode, specify a search string to be used in an SQL query.
```
Comment (callapa) [inline centreon/src/Core/Domain/Configuration/Model/MetaService.php:78]: Not sure we should keep this rule 🤔
In addition, the 120-character limit per line is not respected.

File: .github/workflows/web.yml [hunk 2]
```diff
@@ -264,6 +264,32 @@ jobs:
           path: "centreon/build/phpunit.xml"
           format: "php-junit"
 
+  php-cs-fixer:
+    runs-on: ubuntu-22.04
+    needs: [changes]
```
Comment (tuntoja) [inline .github/workflows/web.yml:269]:
```suggestion
    needs: [changes, get-version]
```

File: .github/workflows/web.yml [hunk 3]
```diff
@@ -264,6 +264,32 @@ jobs:
...
+    if: ${{ needs.changes.outputs.has_backend_changes == 'true' }}
```
Comment (tuntoja) [inline .github/workflows/web.yml:270]:
```suggestion
    if: ${{ needs.changes.outputs.has_backend_changes == 'true' && needs.get-version.outputs.stability != 'stable' }}
```

**Review by tuntoja — CHANGES_REQUESTED** [review]
Add a condition to not trigger this job on workflow runs carried by stable branches.

File: .github/workflows/web.yml [hunk 4]
```diff
@@ -264,6 +264,32 @@ jobs:
...
+          runner: ubuntu-22.04
+
+      - name: Install dependencies
+        uses: ramsey/composer-install@v2
+        with:
+          working-directory: centreon
+          composer-options: "--optimize-autoloader"
+
+      - name: Run of php-cs-fixer
+        run: vendor/bin/php-cs-fixer fix --verbose --dry-run
```
Comment (abuathier) [inline .github/workflows/web.yml:290]:
```suggestion
        run: vendor/bin/php-cs-fixer fix --verbose --dry-run --diff
```

**Review by callapa — DISMISSED** [review]

**Review by adr-mo — DISMISSED** [review]

**Review by kduret — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

**Review by adr-mo — APPROVED** [review]

**Review by callapa — APPROVED** [review]

**Pull request merged**